lms/static/sass/discussion/views/_inline.scss

```diff
@@ -22,7 +22,7 @@
                 .icon {
                     position: absolute;
                     @include left(7px);
-                    top: 17px;
+                    top: 16px;
```

Comment:
This is to position the < correctly in front of "All Posts", which AFAIK is not inside a flexbox element... it's a bit hackish, yeah. I'll take a look and see if anything can be done